content/v1/features/powerflex.md

```diff
@@ -497,6 +497,26 @@ or you could edit logConfig.yaml, and use this command:
 `kubectl apply -f logConfig.yaml`  
 
 and then make the necessary adjustments for LOG_LEVEL and LOG_FORMAT.  
-If LOG_LEVEL or LOG_FORMAT are set to options outside of what is supported, the driver will use the default values of "info" and "text" . 
+If LOG_LEVEL or LOG_FORMAT are set to options outside of what is supported, the driver will use the default values of "info" and "text" .   
```

Comment:
Is there a space at the end?  thats the only change i see here.

Reply:
I think it's showing that I added a new line